polls/tool/src/bundle/org/sakaiproject/poll/bundle/Messages.properties

```diff
@@ -50,7 +50,9 @@ new_poll_option=Option
 new_poll_submit=Save
 new_poll_saveoption=Save and add options
 new_poll_option_title=Options
-new_poll_option_add=Add option
+new_poll_option_add=Add options one by one
+new_poll_option_add_batch=Add options massively
+new_poll_option_batch_info=You can upload an Excel or a CSV file with all the options in the first column, the rest of the columns will be ignored.
```

Comment:
This is a comma splice, which is grammatically incorrect. Either change the comma to a semicolon, or change it to a period and capitalize the "t" in "The".

Reply:
Good